tracer/src/Datadog.Tracer.Native/clr_helpers.h

```diff
@@ -243,8 +243,8 @@ struct AssemblyInfo
     {
     }
 
-    AssemblyInfo(AssemblyID id, shared::WSTRING name, ModuleID manifest_module_id, AppDomainID app_domain_id,
-                 shared::WSTRING app_domain_name) :
+    AssemblyInfo(AssemblyID id, const shared::WSTRING& name, ModuleID manifest_module_id, AppDomainID app_domain_id,
+                 const shared::WSTRING& app_domain_name) :
```

Comment:
FYI, you could have use `std::move` and leave the definition of `app_domain_name` like this